templates/expose.hbs

```diff
@@ -5,7 +5,7 @@
             {{#each _items}}
             <div class="expose-item nth-child-{{@index}}">
                 <div class="expose-item-inner">
-                    <button class="expose-item-button"></button>
+                    <button class="expose-item-button" title="{{#if front}} {{{a11y_text front}}} {{else}} reveal {{/if}}"></button>

```

Comment:
- As well as making text accessible we also need to be mindful about not hard coding language strings.
- Reveal is quite vague - I would default to something like 'Expose button'
- Would an aria label be more appropriate than a title? That way the label would still be present regardless of there being a title or not.
- Language string should be placed in _globals._components._expose.buttonLabel and schema needs updating
- Code wise there shouldn't really be any spaces in between Handlebars helpers otherwise Handlebars will render these. E.g. `{{#if front}}{{{a11y_text front}}}{{else}}reveal{{/if}}`. I know reading is more difficult but the output is tidier.

Reply:
I can try next time I have a window of opportunity.